locust/docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+HOST=${HOST:-http://localhost:8000}
+NUM_USERS=${NUM_USERS:-200}
+SPAWN_RATE=${SPAWN_RATE:-20}
+RUN_TIME=${RUN_TIME:-180}
+PUSHGATEWAY_URL=${LOAD_TEST_PAUSE:-http://prometheus:9091/metrics/job/trialRun}
```

Comment:
This would override the Pushgateway URL being passed in from the controller.

Reply:
I can delete that line. But if the controller injects the `PUSHGATEWAY_UR` environment variable I was expecting that it would overwrite the value no?